tests/containers/forms/LogActivityForm.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import LogActivityForm from '../../../src/containers/forms/LogActivityForm';
+
+describe('<Stats />', () => {
+  it('should render withour crashing', () => {
+    expect(mount.bind(null, <LogActivityForm />)).not.toThrow();
+  });
+  it('should show the <DateField/> component when it has loaded', () => {
+    const wrapper = shallow(<LogActivityForm />);
+    wrapper.setState({ loading: false });
+    const dateComponent = wrapper.find('DateField');
+    expect(dateComponent.length).toEqual(1);
+  });
+  it('should show the <Select/> component when it has loaded', () => {
+    const wrapper = shallow(<LogActivityForm />);
+    wrapper.setState({ loading: false });
+    const selectComponent = wrapper.find('Select');
+    expect(selectComponent.length).toEqual(1);
+  });
+  it('should show the <SingleInput/> component when it has loaded', () => {
+    const wrapper = shallow(<LogActivityForm />);
+    wrapper.setState({ loading: false });
+    const inputComponent = wrapper.find('SingleInput');
+    expect(inputComponent.length).toEqual(1);
+  });
+  it('should show the <TextArea/> component when it has loaded', () => {
+    const wrapper = shallow(<LogActivityForm />);
+    wrapper.setState({ loading: false });
+    const inputComponent = wrapper.find('TextArea');
+    expect(inputComponent.length).toEqual(1);
+  });
+  it('should show the <Buttons/> component when it has loaded', () => {
+    const wrapper = shallow(<LogActivityForm />);
+    wrapper.setState({ loading: false });
+    const inputComponent = wrapper.find('Buttons');
+    expect(inputComponent.length).toEqual(2);
+  });
```

Comment:
Just a  suggestion: Since it is highly advised to have all declarations within a function to happen at the top, I will highly recommend we have only one `const` and then make our assertion as:

```
expect(wrapper.find('Buttons').length).toEqual(2);
```